tests/dmodel/testContent.js

```diff
@@ -22,7 +22,7 @@ describe('Content Object Model', function () {
 
     it('successfully creates new object from properties', function () {
         contentObject = DModel.Content.new_from_props({
-            ekn_id : 'ekn:text_editors/Emacs',
+            id: 'ekn:text_editors/Emacs',
```

Comment:
Doesn't this violate the ekn URI syntax?

Reply:
I added a commit to fix that.